plugin/pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -49,31 +47,34 @@ func ImageLocalityPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *scheduler
 // 1. Split image size range into 10 buckets.
 // 2. Decide the priority of a given sumSize based on which bucket it belongs to.
 func calculateScoreFromSize(sumSize int64) int {
-	var score int
 	switch {
+	// score == 0 means none of the images required by this pod are present on this
```

Comment:
I think you should update this comment and replace 'score == 0 means ...' with 'zero means...'. You should also move it right before the return.